index.js

```diff
@@ -425,15 +424,11 @@ const plugin = fp(async function (app, opts) {
     }
 
     if (cached && cached.jit !== null) {
-      const res = await cached.jit.query(root, context, variables || {})
+      const execution = await cached.jit.query(root, context, variables || {})
 
-      if (res.errors) {
-        const { response: { data, errors } } = errorFormatter(res)
-        res.data = data
-        res.errors = errors
-      }
+      maybeFormatErrors(execution, reply)
 
-      return res
+      return execution
```

Comment:
Would it be neater to make the API `return formatResult(execution, reply)`, and update `maybeFormatErrors` to `formatResult`, with an extra `return execution` at the end?